ksp_plugin_adapter/localization/en-us.cfg

```diff
@@ -57,11 +57,13 @@ Localization {
     #Principia_ReferenceFrameSelector_Name_BodyCentredNonRotating = <<1>>-Centred Inertial
     #Principia_ReferenceFrameSelector_Name_BodyCentredParentDirection = <<1>>-<<3>>-Orbit
     #Principia_ReferenceFrameSelector_Name_BodySurface = <<1>>-Centred <<1>>-Fixed
+    #Principia_ReferenceFrameSelector_Name_RotatingPulsating = <<3>>-<<1>> Lagrange
 
     #Principia_ReferenceFrameSelector_Abbreviation_Target.en-us = Tgt <<2>>O
     #Principia_ReferenceFrameSelector_Abbreviation_BodyCentredNonRotating.en-us = <<2>>CI
     #Principia_ReferenceFrameSelector_Abbreviation_BodyCentredParentDirection.en-us = <<2>><<4>>O
     #Principia_ReferenceFrameSelector_Abbreviation_BodySurface.en-us = <<2>>C<<2>>F
+    #Principia_ReferenceFrameSelector_Abbreviation_RotatingPulsating.en-us = <<4>><<2>>L
```

Comment:
Not the same order as BCBD.  Is that intended?

Reply:
Yes.